entries/s/scu.edu.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Santa Clara University": {
+    "domain": "scu.edu",
+    "tfa": [
+      "sms",
+      "custom-software",
+      "custom-hardware",
```

Comment:
From reading the [FAQ](https://www.scu.edu/technology/get-connected/duo/2fa-faqs/) it sounds like U2F/FIDO 2 keys are the only hardware 2fa methods supported by SCU. If that's correct then the custom-hardware method can be removed:
```suggestion
```